puppet/zulip/files/supervisor/conf.d/thumbor.conf

```diff
@@ -0,0 +1,14 @@
+[program:zulip-thumbor]
+command=thumbor --port=9995 --conf=/home/zulip/deployments/current/zthumbor/thumbor.conf
+user=zulip
+directory=/home/zulip/deployments/current/
+autostart=true
+autorestart=true
+startretries=3
+stopsignal=TERM
+stdout_logfile=/var/log/zulip/thumbor.stdout.log
+stdout_logfile_maxbytes=1MB
+stdout_logfile_backups=10
+stderr_logfile=/var/log/zulip/thumbor.stderr.log
+stderr_logfile_maxbytes=1MB
+stderr_logfile_backups=10
```

Comment:
I would follow the convention of our other supervisord files for stdout/stderr, namely, putting both stderr and stdout in the same file in `/var/log/zulip/thumbor.log` via `redirect_stderr`, and having the log rotation be 20MB x 3.